uni/unidoc/Makefile

```diff
@@ -30,15 +26,14 @@ help:
 #    commonly used suffixes.
 #
 %.u:	%.icn
-	$(UNICON) $(UFLAGS) -c $?
+	unicon -c $?
```

Comment:
Any reason to use Unicon instead if the variable being replaced ?

Reply:
I didn't see where $(UNICON) was defined anywhere.  Perhaps it's defined if this Makefile is invoked from the main
Unicon make (i.e. via ${UNICON}/make), but a direct make in the uni/unidoc directory fails.

Reply:
All unicon/uni directories get their definitions from the top level Makefile.uni 

BASE=../..
include $(BASE)/Makedefs.uni

Reply:
Ah - the Makefile in uni/unidoc is missing the lines:

BASE=../..
include $(BASE)/Makedefs.uni

lines at the top.